internal/cli/file/file.go

```diff
@@ -118,6 +119,43 @@ func visit(dir string) ([]string, error) {
 	return files, err
 }
 
+// Copy a directory recursively
+func CopyDir(src, dst string) error {
+	return filepath.Walk(src, func(path string, info os.FileInfo, err error) error {
+		if err != nil {
+			return err
+		}
+		relPath, err := filepath.Rel(src, path)
+		if err != nil {
+			return err
+		}
+		targetPath := filepath.Join(dst, relPath)
+
+		if info.IsDir() {
+			return os.MkdirAll(targetPath, os.ModePerm)
+		}
+		return CopyFile(path, targetPath)
+	})
+}
+
+// Copy a file
+func CopyFile(src, dst string) error {
```

Comment:
instead of `CopyFile` it can simply be `Copy`. So when used `file.Copy` you can understand you are copying a file